tests/graphql/test-nested-value.graphql

```diff
@@ -0,0 +1,75 @@
+#> server-sdl
```

Comment:
isn't it too many types to do the testing? it could be simplified I think.

Reply:
agree! done :smile: